administration/src/cards/extensions/BirthdayExtension.tsx

```diff
@@ -57,7 +57,7 @@ const BirthdayExtension: Extension<BirthdayExtensionState> = {
     },
   }),
   isValid: ({ birthday }: BirthdayExtensionState) => {
-    if (birthday === null) {
+    if (birthday == null) {
```

Comment:
:wrench: Please revert this. I don't understand why the checks are still passing for this?

Reply:
Why? I wanted to get false if birthday is null AND if birthday is undefined. How to do it better?

Reply:
You can just do
``` ts
if (!birthday) {
...
}
```
Using only two equals `==` only performs loose type checking and therefore does implicit js type logic. Therefore, we only want strict equal `===`.

See e.g. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Equality_comparisons_and_sameness or https://stackoverflow.com/questions/16633947/in-javascript-is-it-better-to-use-two-equal-signs-or-three-when-comparing-varia

Reply:
Ok, thx 🙈